src/chain/member_info.rs

```diff
@@ -7,18 +7,40 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 /// Information about a member of our section.
+
+pub type AgeCounter = u32;
```

Comment:
I'd suggest to be more type-safe here: create a proper type for this:
```rust
#[derive(Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
pub struct AgeCounter(u32);
```
and 
```rust
impl AgeCounter {
    pub fn age(&self) -> u8 {
        f64::from(self.0).log2() as u8
    }

    pub fn increment(&mut self) {
        self.0 += 1; // or consider using saturating_add, just in case
    }
}
```

Reply:
And also
```rust
impl Default for AgeCounter {
    fn default() -> Self {
        Self(MIN_AGE)
    }
}
```